app/src/main/java/com/github/sdp/ratemyepfl/adapter/ReviewableAdapter.kt

```diff
@@ -3,36 +3,42 @@ package com.github.sdp.ratemyepfl.adapter
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.Button
 import android.widget.Filter
 import android.widget.Filterable
+import android.widget.LinearLayout
 import android.widget.TextView
 import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
 import com.github.sdp.ratemyepfl.R
 import com.github.sdp.ratemyepfl.model.items.Course
```

Comment:
I guess it is not useful ;-)

Reply:
You can run it on the files you change :)

Reply:
Yes but I can't remember them :(